packages/reporting-server/models/auth_events.py

```diff
@@ -0,0 +1,21 @@
+from tortoise import fields, models
+from tortoise.contrib.pydantic import pydantic_model_creator
+
+
+class AuthEvents(models.Model):
+    id = fields.IntField(pk=True)
+    # user = fields.ForeignKey(User, null=True)
+    username = fields.TextField(null=True)
```

Comment:
Should we use `TextField` or `CharField`? I'm guessing `CharField` is more performant.

Reply:
Yeah, you're right. Updating